### ISSUE TITLE
Deprecation Notice: Class Hackzilla\PasswordGenerator\Tests\RandomGenerator\HybridPasswordGeneratorTest

### ISSUE THREAD

**Issue by tonix-tuft** [hackzilla/password-generator#32]
Hello,

as I have updated Composer to version `1.10.5 2020-04-10 11:44:22`, I get the following warning now when I run the following command:

`composer self-update && composer install --no-dev && composer update --lock`

```
Deprecation Notice: Class Hackzilla\PasswordGenerator\Tests\RandomGenerator\HybridPasswordGeneratorTest 
```

**Comment by hackzilla** [comment]
Was there any more to that deprecation notice?
I can't reproduce it.

**Comment by tonix-tuft** [comment]
@hackzilla 

Thank you for your reply. I am sorry, I thought I have posted everything.

Here is the complete line Composer is outputting:

```
>> Deprecation Notice: Class Hackzilla\PasswordGenerator\Tests\RandomGenerator\HybridPasswordGeneratorTest
located in ./vendor/hackzilla/password-generator/Tests/RandomGenerator/NoPasswordGeneratorTest.php
does not comply with psr-4 autoloading standard.
It will not autoload anymore in Composer v2.0. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
```

**Comment by tonix-tuft** [comment]
It refers to the PSR-4 autoloading mechanism of Composer.

**Comment by hackzilla** [comment]
Thanks.

Can you confirm which version of password-generator you are using?

**Comment by hackzilla** [comment]
Actually, I remember this.

I'm sure it is fixed somewhere in master, and just hasn't been tagged.

**Comment by hackzilla** [comment]
I've just checked.
It was fixed in 1.5.0 by @SunMar

**Comment by tonix-tuft** [comment]
Great! I run `composer update` and the warning disappeared. Closing this. Thank u!
